### PR TITLE
chore: don't unmount planks

### PR DESCRIPTION
**Pull request by richburdon**

### CODE REVIEW

File: packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx [hunk 1]
```diff
@@ -33,27 +34,28 @@ import { useLayout } from '../LayoutContext';
...
   showHintsFooter: boolean;
-  overscroll: Overscroll;
-  flatDeck?: boolean;
   toasts: ToastSchema[];
-  onDismissToast: (id: string) => void;
   // TODO(burdon): Rename planks or just items?
   layoutParts: LayoutParts;
   attention: Attention;
+  // TODO(burdon): Remove.
+  flatDeck?: boolean;
```
Comment (thure) [inline packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:42]: This is a setting just like wallpaper… why remove?

File: packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx [hunk 2]
```diff
@@ -149,73 +154,64 @@ export const DeckLayout = ({
...
+              </Deck.Root>
+            </div>
           </Main.Content>
         )}
 
+        {/* TODO(burdon): Remove? */}
         {/* Note: This is not Main.Content */}
         <Main.Content role='none' classNames={['fixed inset-inline-0 block-end-0 z-[2]']}>
           <Surface role='status-bar' limit={1} />
         </Main.Content>
```
Comment (thure) [inline packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:211]: The status bar is how we invoke the status panel, the discord link, and the feedback popover. We should definitely discuss before removing.

**Review by thure — APPROVED** [review]
Looks fine to me, just some remarks about the todo’s this adds.

File: packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx (hunk 1 above)
Comment (richburdon) [reply packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:42]: old TODO (i removed the thing!)

File: packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx [hunk 3]
```diff
@@ -149,73 +154,64 @@ export const DeckLayout = ({
...
-        {isEmpty && (
+        {parts.length === 0 && (
           <Main.Content>
             <ContentEmpty />
           </Main.Content>
         )}
 
         {/* Solo/deck mode. */}
-        {!isEmpty && (
+        {parts.length !== 0 && (
```
[the remark this answers is not in the document]
Comment (richburdon) [reply packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:174]: i find the ternary ifs more confusing.

File: packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx (hunk 2 above)
Comment (richburdon) [reply packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:211]: oh i didn't know what this is. there's some other old things here -- like the hints.
I thought this was the "old" main.

**Pull request merged**